docs/resources/recipientLists.md

```diff
@@ -2,63 +2,30 @@
 
 This library provides easy access to the [Recipient Lists](https://developers.sparkpost.com/api/recipient-lists) Resource.
 
+*Note: All methods return promises and accept an optional last argument callback. [Read about how we handle callbacks and promises](/docs/async.md).*
+
 ## Methods
-* **all([callback]) &rarr; `{Promise}`**<br />
+* **list()**<br />
   List a summary of all recipient lists.
-  * `callback` -  executed after task is completed if provided*
-    * standard `callback(err, data)`
-    * `err` - any error that occurred
-    * `data` - full response from request client
-* **find(options[, callback]) &rarr; `{Promise}`**<br />
+
+* **get(id[, options)`**<br />
```

Comment:
There's an extra back tick and you need a close ] after options
get(id[, options])